main.js

```diff
@@ -44,11 +44,57 @@ for (let i = 0; i < arr2.length; i++) {
   }
 }
 
-//4.
+//5.
 const arr3 = [2, 5, 8, 15, 0, 6, 20, 3];
 
 for (let i = 0; i < arr3.length; i++) {
   if (arr3[i] % 2 === 0) {
     console.log(arr3[i]);
   }
 }
+
+//lesson-3
+//1.
+function palindrome(str) {
+  str = str.toLowerCase();
+  return str === str.split('').reverse().join('');
+}
```

Comment:
it will be great if your functions be clean. don't change your arguments, create new variable for that.

Reply:
TY. All fixes wil be in lesson-4.